lib/StaticNarrative/exporter/exporter.py

```diff
@@ -27,34 +27,37 @@
 from StaticNarrative.narrative_ref import NarrativeRef
 
 NARRATIVE_TEMPLATE_FILE = "narrative.tpl"
+OUTPUT_HTML_FILE = "narrative.html"
 
 
 class NarrativeExporter:
     """Class for exporting Static Narratives."""
 
     def __init__(
         self: "NarrativeExporter",
-        exporter_cfg: dict[str, str],  # config object
+        config: dict[str, str],  # config object
         user_id: str,
         token: str,
         use_set_api: int = 1,
         debug: bool = False,
     ) -> None:
         """Initialise the Narrative Exporter."""
-        self.exporter_cfg = exporter_cfg
-        self.ws_client = Workspace(url=exporter_cfg["workspace-url"], token=token)
+        self.config = config
```

Comment:
i found the `exporter_cfg` name a bit confusing since it's the same as the config elsewhere